src/test/java/com/hocs/test/pages/decs/Workstacks.java

```diff
@@ -832,6 +832,35 @@ public void assertHomeSecretarySymbolVisibleForCase(String caseReference) {
         assertThat(homeSecSymbol.isCurrentlyVisible(), is(true));
     }
 
+    public void assertSecretariatClearanceRequestedDueDate() {
+        waitForWorkstackToLoad();
+        String inputClearanceDueDate = sessionVariableCalled("clearanceDueDate");
+        String currentCaseRef = sessionVariableCalled("caseReference");
+        WebElementFacade currentStageCell = findBy("//a[text()='" + currentCaseRef + "']/parent::td/following-sibling::td[1]");
+        String dueDate = currentStageCell.getText().split("due ")[1];
+        assertThat(inputClearanceDueDate.equals(dueDate), is(true));
```

Comment:
I think a lot of this could have been covered by the method assertSpecifiedColumnContainsValueForCurrentCase ?

Reply:
Replaced unnecessary steps with helper method